README.md

```diff
@@ -16,3 +16,5 @@ The MIT License (MIT)
 
 Copyright (c) 2016 Katrina Owen, _@kytrinyx.com
 
+### Purescript icon
+The Purescript icon is assumed to be released under the MIT license, like the project itself, though its author is not known. We have adapted it, changing the colour scheme, for use on Exercism.
```

Comment:
The logo license information is listed [here](https://github.com/purescript/logo/blob/master/README.md).